chain/vm/vm.go

```diff
@@ -180,7 +179,6 @@ func NewVM(base cid.Cid, height abi.ChainEpoch, r Rand, maddr address.Address, c
 		cst:         cst,
```

Comment:
Can't leave a comment on the exact line, but above on [168](https://github.com/filecoin-project/lotus/pull/1524/files#diff-b2eed56186dd5c40cd534220ec40c1a9R168) `maddr` can  be removed from `NewVM()` as well (unless I am missin something).

Reply:
Done :)